test/units/testsuite-46.service

```diff
@@ -3,8 +3,8 @@
 Description=TEST-46-HOMED
 Wants=getty-pre.target
 Before=getty-pre.target
-Wants=systemd-homed.service
-After=systemd-homed.service
+Requires=systemd-homed.service systemd-userdbd.service
```

Comment:
makes sense, but please only pull in the socket unit, not the service. THat should be enough, and tests that the thing is correctly socket-activated. i.e. `systemd-userdbd.socket)